dashboard/test/state/build_test.dart

```diff
@@ -45,6 +47,8 @@ void main() {
             ..repository = 'flutter'
         ]),
       );
+
+      FlutterAppIconsPlatform.instance = FakeFlutterAppIcons();
```

Comment:
You could just do this at the start of `main` instead of `setUp`, since it doesn't need to run per test, but it's also harmless to recreate the fake.

Reply:
Thanks, but I'll keep it in setUp. In the off chance this stops being hermetic, it won't start to cause flakes